rca/enquire_to_study/views.py

```diff
@@ -33,6 +36,32 @@ def post_qs(self, form_data):
     def create_form_submission(self, form):
         form.save()
 
+    def send_user_email_notification(self, form):
+        enquiry_form_settings = EnquireToStudySettings.objects.first()
```

Comment:
Is this site single or multi-site? `EnquireToStudySettings.objects.first()` will only return the correct settings for one site in a multi-site setup, in which case you'd use https://docs.wagtail.io/en/stable/reference/contrib/settings.html#using-in-python

Reply:
Thanks, yea this was lazy on my part. It's one site but in the future there may be more. Fixed in https://github.com/torchbox/rca-wagtail-2019/pull/586/commits/68ca60e026ee71f25662f224b53e24e6ca8d593d